tailwind.config.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
         wnv: "#CAFFBF",
         yf: "#FFD6A5",
         mayv: "#c5a3ff",
+        link: "#0000EE",
```

Comment:
This is actually the recommendation from the HTML spec which is kind of interesting.
https://html.spec.whatwg.org/multipage/rendering.html#phrasing-content-3